data/ritika728.json

```
@@ -0,0 +1,23 @@
+{
+	"name": "Ritika Malik",
+	"displayStatsPublic": true,
+	"type": "personal",
+	"bio": "Front-end Developer | Open Source Enthusiast",
+	"avatar": "https://github.com/ritika728.png",
+	"links": [{
+			"name": "Follow me on Linkedin",
+			"url": "www.linkedin.com/in/ritika-malik-must/",
```

You probably want to use `https` instead, as it could lead to an unwanted link.
```suggestion
			"url": "https://www.linkedin.com/in/ritika-malik-must/",
```